server/src/internalClusterTest/java/org/opensearch/discovery/MasterDisruptionIT.java

```diff
@@ -91,7 +91,7 @@ public void testMasterNodeGCs() throws Exception {
 
         logger.info("waiting for nodes to de-elect master [{}]", oldClusterManagerNode);
         for (String node : oldNonClusterManagerNodesSet) {
-            assertDifferentMaster(node, oldClusterManagerNode);
+            assertDifferentClusterManager(node, oldClusterManagerNode);
         }
 
         logger.info("waiting for nodes to elect a new master");
```

Comment:
Are we changing the logs also in this PR? 
`new master -> new cluster manager`

Reply:
Thank you for your review! 👍👍
The `master` word in the log is changed in the other PR https://github.com/opensearch-project/OpenSearch/pull/2521/files#diff-d5423ad6b9197f9d92b8c463df4320e6e2c3ca85768fde53c7e8d01f5945732e.
This PR is used to replace the `master` terminology in `server/src/test` directory, so I changed its name in another PR.
While the reason that the file in `server/src/internalClusterTest` got changed is some methods are defined in `server/src/test` but got reference in `server/src/internalClusterTest`, so I changed the name together in this PR for convenience.